catalog/app/components/Preview/types.js

```diff
@@ -4,19 +4,13 @@ import tagged from 'utils/tagged'
 ParquetMeta: {
```

Comment:
i think you can safely remove `ParquetMeta` *definition* from here, bc it's explicitly present in the `Tabular` loader module